pipeline/test/update-prismic-snapshots.ts

```diff
@@ -91,7 +93,10 @@ const updateEventDocumentSnapshots = async (client: Client) => {
     docs.map(doc => {
       const docJson = JSON.stringify(doc, null, 2);
       return fs.writeFile(
-        path.resolve(dataDir, `${doc.id}.${doc.type}.json`),
+        path.resolve(
+          dataDir,
+          `${doc.id}.${doc.type === 'exhibitions' ? 'events' : doc.type}.json`
+        ),
```

Comment:
Made sense to me to put them under `.events.json`, but we can always change it later on